src/test/java/gui/TestEditRulePopup.java

```diff
@@ -0,0 +1,17 @@
+package test.java.gui;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import main.java.gui.EditRulePopup;
+
+class TestEditRulePopup {
+	@Test

```

Comment:
please explain what's being done in the test, namely that you're validating the frame size to ensure the popup is being shown